Dockerfile

```diff
@@ -60,6 +60,11 @@ ENV PATH /opt/freesurfer/bin:/opt/freesurfer/fsfast/bin:/opt/freesurfer/tktools:
 ENV PYTHONPATH=""
 RUN echo "cHJpbnRmICJrcnp5c3p0b2YuZ29yZ29sZXdza2lAZ21haWwuY29tXG41MTcyXG4gKkN2dW12RVYzelRmZ1xuRlM1Si8yYzFhZ2c0RVxuIiA+IC9vcHQvZnJlZXN1cmZlci9saWNlbnNlLnR4dAo=" | base64 -d | sh
 
+# make freesurfer python scripts python3 ready
+RUN 2to3-3.4 -w $FREESURFER_HOME/bin/aparcstats2table
+RUN 2to3-3.4 -w $FREESURFER_HOME/bin/asegstats2table
+RUN 2to3-3.4 -w $FREESURFER_HOME/bin/*.py
```

Comment:
OK but my cluster does not have singularity installed and if I install it myself with my user privileges I get this error when I try to execute the image:
```
-bash-4.1$ ./bids_example-2016-10-27-709845fcdcd0.img 
ERROR  : Singularity must be executed in privileged mode to use images
ABORT  : Retval = 255
```
Anyway it is not a big problem for me because I can easily modify this part of the script myself since everything else is working fine without the need to be in a container.